cmd/cmd.go

```diff
@@ -8,8 +8,8 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
-	"github.com/alexandrst88/terraform-variables-generator/pkg/generator"
-	"github.com/alexandrst88/terraform-variables-generator/pkg/utils"
+	"github.com/marceloalmeida/terraform-variables-generator/pkg/generator"
```

Comment:
?

Reply:
It was to be able to perform `go get...` on multiple places and build using my changes, just reverted.